software/gui/gui.sh

```diff
@@ -169,75 +120,102 @@ install_linux() {
           qml-module-qtquick-controls2 \
           qml-module-qtmultimedia \
           pulseaudio \
+          python-pip \
```

Comment:
Deprecated for my distro, recommanded `python3-pip`

Reply:
Indeed. Updated.